src/lavozScrapper.py

```diff
@@ -118,8 +97,8 @@ def scrap_ids(self):
                 if i == 0:
                     page_part = ""
 
-                ##https://clasificados.lavoz.com.ar/buscar/inmuebles?filters%5Bvendedor%5D%5B0%5D=particular&page=1
-                search_url_inmu = "https://clasificados.lavoz.com.ar/buscar/inmuebles?filters%5Bvendedor%5D%5B0%5D=" + publisher_type + page_part
+                search_url_inmu = 'https://clasificados.lavoz.com.ar/buscar/inmuebles?filters={{"vendedor":{publisher_types}}}{page_part}'.format(
```

Comment:
this is wrong. You will be scrapping for each publisher types all publisher types.... 
Opcion 1) Remove the for and do it once
Opcion 2) Replace {{"vendedor":{publisher_types}}} -> {{"vendedor":[{publisher_type}]}}

In the option 1 you will scrap 2 pages for each publisher type, in the option 2, you will scrap 2 pages including all the publisher types.